gincmd/uploadcmd.go

```diff
@@ -34,15 +35,27 @@ func upload(cmd *cobra.Command, args []string) {
 			break
 		}
 	}
-
+	fmt.Println(args)
 	paths := args
 	if len(paths) > 0 {
 		commit(cmd, paths)
 	}
 
 	uploadchan := make(chan git.RepoFileStatus)
 	go gincl.Upload(paths, remotes, uploadchan)
-	formatOutput(uploadchan, 0, jsonout)
+	if verbose == true {
+		var cmd_spec_var []string
```

Comment:
We use camelCase for variable names
```suggestion
		var cmdSpecVar []string
```